schema_to_cr.go

```diff
@@ -36,6 +36,7 @@ func main() {
 	makeCrdPtr := flag.Bool("makecrd", false, "Boolean option to choose whether to generate a new CRD file (optional; default false)")
 	omitPtr := flag.String("omit", "", "Option to omit creating CR entries for types starting with the given string(s). Multiple strings should be comma-separated - e.g. \"read,list\" (optional).")
 	crNamespacePtr := flag.String("crnamespace", "", "Option to use a different namespace for the CRs if {{ .Release.Namespace }} is not desired")
+	guardPtr := flag.Bool("guard", true, "Boolean option to choose whether to include the guard condition in the CR and CRD files (optional; default true)")
```

Comment:
default true cli args are annoying to deal with. Maybe call it "skip-guard", false or remove it entirely. Nobody will be able to develop locally with these CRs without guard blocks around them.